src/lib/utils.ts

```diff
@@ -13,14 +13,17 @@ export function cn(...inputs: ClassValue[]) {
 }
 
 export function extractTitleFromMessage(message: string) {
+  if (message.includes("review codegate")) {
+    console.log("message is");
+    console.log(JSON.stringify(message, null, 2));
+  }
```

Comment:
🤔 why we need to log this? usually it is better to avoid console log for security reason